istio/install-istio.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const (
-	repoURL     = "https://api.github.com/repos/istio/istio/releases/latest"
+	repoURL     = "https://api.github.com/repos/istio/istio/releases/24270913"
```

Comment:
This is not latest isitio installation URL.
If you want to specify only 1.5, i would suggest you fix the tags instead of fixing/hardcoding the release URL.

Reply:
If you see the install-istio.go , We are not using Tags & have just described it. We are right now fetching the latest istio version from the installation URL only.

Reply:
1. This url doesn't represent latest release, it is 1.5
2. We are not using tags because we always fetch latest release, if we want to revert to old release, we should use tags api of release so that it's clear which release is being targeted ( This should be avoided anyhow, if release needs to be fixed, fix the url of istio tar file)

Reply:
1. This url doesn't represent latest release, it is 1.5
2. We are not using tags because we always fetch latest release, if we want to revert to old release, we should use tags api of release so that it's clear which release is being targeted ( This should be avoided anyhow, if release needs to be fixed, fix the url of istio tar file)

Reply:
Also please refer https://github.com/layer5io/meshery-istio/issues/85#issuecomment-632963393

Reply:
> Also please refer [#85 (comment)](https://github.com/layer5io/meshery-istio/issues/85#issuecomment-632963393)

It depends on @leecalcote if we want to introduce a arch field just for sake of updating to 1.6, IMO it wont be wise choice.

Reply:
Hey @nveenjain @kushthedude 
It's working all right (after changing the suffix URL) when I use https://api.github.com/repos/istio/istio/releases/27703087
but not when I use https://api.github.com/repos/istio/istio/releases/latest which is strange because the 27703087 is the ID for the latest (1.6.3) release. Am I doing something wrong?
Sorry If it is a stupid question.

Reply:
https://github.com/layer5io/meshery-istio/issues/87#issuecomment-638607006
^^ @harsh-not-haarsh